src/Moment/MomentGenerator.cc

```diff
@@ -110,7 +112,7 @@ void MomentGenerator::SetMomentAxis(const CARTA::MomentRequest& moment_request)
     } else if (moment_request.axis() == CARTA::MomentAxis::STOKES) {
         _axis = _stokes_axis;
     } else {
-        std::cerr << "Do not support the moment axis: " << moment_request.axis() << std::endl;
+        spdlog::error("Do not support the moment axis: {}", moment_request.axis());
```

Comment:
Description should be changed to `"Unsupported moment axis: "`.

Reply:
:+1: